content/components/navigation/pagination.md

```diff
@@ -45,50 +145,69 @@ When using the `.is-disabled` classes to create pagination links that appear to
 ### Positioning
 The pagination component is left-aligned by default, but you can easily center or right-align it using the `rvt-pagination--center` or `rvt-pagination--right` modifier classes.
```

Comment:
I notice here that the classes are referenced without a period at the beginning, but in other places (ex. line 181) a period is still used. Not sure which way is preferred, but picking one for consistency would probably be best.

Reply:
Looks like Scott has already has us covered on this one. 😉 

https://github.com/indiana-university/rivet-source/wiki/Editorial-style-guide#code

I'll update that.

Reply:
🙌